pkiinit/pkiInit.go

```diff
@@ -29,31 +32,47 @@ type exitCode struct {
 	exit
 }
 
+type optionDispatcher interface {
+	run() (int, error)
+}
+
+type pkiInitOptionDispatcher struct {
+	optionDispatcher
```

Comment:
could be simpler by removing this object and changing newOptionDispatcher to 
func newOptionDispatcher() optionDispatcher {
	return &pkiInitOptionDispatcher{}
}